imports/plugins/core/catalog/server/no-meteor/utils/createCatalogProduct.js

```diff
@@ -252,10 +252,11 @@ export default async function createCatalogProduct(product, context) {
   const catalogProduct = await xformProduct({ collections, product, shop, variants });
 
   // Apply custom transformations from plugins.
-  getFunctionsOfType("publishProductToCatalog").forEach((customPublishFunc) => {
+  for (const customPublishFn of getFunctionsOfType("publishProductToCatalog")) {
```

Comment:
@aldeed you mentioned writing up a few test for this bug, what should I test for here? I'm looking to make sure each `customPublishFn` has been called before we return out of this func?